packages/components/pie-modal/src/defs.ts

```diff
@@ -105,6 +105,12 @@ export interface ModalProps extends RTLComponentProps {
      * The position of the modal; this controls where it will appear on the page.
      */
     position: typeof positions[number];
+
+    ariaLabels?: {
```

Comment:
how does snacks do this? I believe we wanted to take inspiration from them. @raoufswe @fernandofranca?

Reply:
The ticket mentioned something about making this shareable but given the labels are specific to back, close and loading I didn't think making it shareable made sense, saying that though if there is a better way happy to make that change.

Reply:
I would name them to be concise

```suggestion
    aria?: {
    close?: string;
    back?: string;
    loading?: string;
    }
```

Reply:
Fixed.